mailpoet/lib/Newsletter/Scheduler/AutomaticEmailScheduler.php

```diff
@@ -41,19 +43,19 @@ public function scheduleAutomaticEmail($group, $event, $schedulingCondition = fa
     }
   }
 
-  public function scheduleOrRescheduleAutomaticEmail($group, $event, $subscriberId, $meta = false) {
-    $newsletters = $this->scheduler->getNewsletters(Newsletter::TYPE_AUTOMATIC, $group);
+  public function scheduleOrRescheduleAutomaticEmail(string $group, string $event, int $subscriberId, array $meta): void {
```

Comment:
`$meta` was made a mandatory argument here, deliberately or by mistake?

Reply:
I found only one usage of this method when the `meta` parameter is filled, so I changed it to mandatory because it makes sense to me.